easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cole-trapnell-lab.github.io/cufflinks/'
+description = """Transcript assembly, differential expression, and differential regulation for RNA-Seq"""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/']
+
+dependencies = [
+    # issues with boost > 1.55, see https://github.com/cole-trapnell-lab/cufflinks/issues/3
+    ('Boost', '1.55.0', '-Python-2.7.9'),
+    ('SAMtools', '0.1.19'),
+    ('Eigen', '3.2.3'),
```

Comment:
I'm just sticking with the same version as intel-2015a, we could move them both forward if it makes sense...